src/test/java/org/sagebionetworks/bridge/exporter/util/BridgeExporterUtilTest.java

```diff
@@ -202,4 +217,123 @@ public void shouldConvertFreeformTextToAttachment() {
                 .withRevision(1).build();
         assertFalse(BridgeExporterUtil.shouldConvertFreeformTextToAttachment(other, "other"));
     }
+
+    @Test
+    public void canConvertToColumnList() {
+        final String testColumnModelName1 = "test_column_model_1";
+        final String testColumnModelName2 = "test_column_model_2";
+
+        List<ColumnModel> testColumnList;
+
+        ImmutableList.Builder<ColumnModel> columnListBuilder = ImmutableList.builder();
+
+        ColumnModel testModel1 = new ColumnModel();
+        testModel1.setName(testColumnModelName1);
+        testModel1.setColumnType(ColumnType.STRING);
+        testModel1.setMaximumSize(36L);
+        columnListBuilder.add(testModel1);
+
+        ColumnModel testModel2 = new ColumnModel();
+        testModel2.setName(testColumnModelName2);
+        testModel2.setColumnType(ColumnType.STRING);
+        testModel2.setMaximumSize(48L);
+        columnListBuilder.add(testModel2);
+
+        testColumnList = columnListBuilder.build();
+
+        List<ColumnDefinition> testColumnDefinitions;
+
+        ImmutableList.Builder<ColumnDefinition> columnDefinitionBuilder = ImmutableList.builder();
+
+        ColumnDefinition testDefinition1 = new ColumnDefinition();
+        testDefinition1.setName(testColumnModelName1);
+        testDefinition1.setColumnType(ColumnType.STRING);
+        testDefinition1.setMaximumSize(36L);
+        columnDefinitionBuilder.add(testDefinition1);
+
+        ColumnDefinition testDefinition2 = new ColumnDefinition();
+        testDefinition2.setName(testColumnModelName2);
+        testDefinition2.setColumnType(ColumnType.STRING);
+        testDefinition2.setMaximumSize(48L);
+        columnDefinitionBuilder.add(testDefinition2);
+
+        testColumnDefinitions = columnDefinitionBuilder.build();
+
+        assertEquals(BridgeExporterUtil.convertToColumnList(testColumnDefinitions), testColumnList);
+    }
+
+    @Test
+    public void canGetRowValuesFromRecordBasedOnColumnDefinition() {
```

Comment:
We need a test case with name but not ddbName, to make sure we're testing that use case.